### ISSUE TITLE
Utility measure: count rows for exact column values

### ISSUE THREAD

**Issue by yoid2000** [gda-score/anon-utility#4]
For the second utility measure, lets do the same thing as #2, but this time measuring the count of rows instead of the count of distinct users.

So, in essence, just like #2 except we'll do `count(*)` instead of `count(distinct uid)`

Probably just update the existing code base with an option of what to count.

**Comment by srnb** [comment]
I have added one more parameter named `measureParam` in `inputParameters` file to construct a query based on the parameter mentioned. For Ex:  Based on whether `uid`  or *(all).

**Comment by yoid2000** [comment]
You said:

>I have made the code changes.I have taken results only for single column. For double column if takes long time(gda_scihub and gda_census0) and server fails to after some time.  Please find the error log and also the output JSON log for based on query count(distinct) and count*.

Certainly with census we can't possibly take every pair of rows. Probably in general we should be smarter about how much we measure ... random sample or something. Let me think about this.

By the way, it should be the case that class `gdaAttack` is caching the results of queries. So if you run a given measure a second time, all of the queries that succeeded in the previous run should come right out of the cache and complete very fast. However I haven't tested this thoroughly. Have you been able to notice if this works or not?

**Comment by srnb** [comment]
Yes, even I have noticed that cache works for already executed queries. But several queries for particular columns takes a long time to get the result back and fails. Meanwhile, I will have a look at the caching part.

**Comment by yoid2000** [comment]
If the cache is surely working, then don't bother to look at it.

The long queries failing on aircloak is a different sort of problem. I ran
the query for which you got an error, and it (eventually) completed. So not
sure what went wrong.

By the way, I'm rethinking a bit how we'll do these utility measures to
make them more efficient and meaningful. Should have something for you in a
day or two. In the meantime, you probably saw that I made a new issue for
you.
